sd-dev/bin/begin.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import os
+import requests
+import subprocess
+import shutil
+
+def parse_args():
+    """
+    Handle CLI args.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--commit",
+        default=False,
+        required=True,
+        action="store",
+        help="Git commit to build from",
+    )
+    args = parser.parse_args()
+    return args
+
+def run(commit):
+    owner = "freedomofpress"
+    repo = "securedrop-workstation"
+
+    working_dir = "/var/lib/sdci-ci-runner"
+    subprocess.run(["sudo", "mkdir", "-p", working_dir])
+    subprocess.run(["sudo", "chown", "user:user", working_dir])
+
+    workspace = f"{repo}_{commit}"
+
+    # Remove copy of this repo if it exists
+    if os.path.exists(f"{working_dir}/{workspace}"):
+        shutil.rmtree(f"{working_dir}/{workspace}")
+
+    # Clone and checkout that relevant commit
+    subprocess.check_call(["git", "clone", f"https://github.com/{owner}/{repo}", f"{working_dir}/{workspace}"])
+    subprocess.check_call(["git", "checkout", commit], cwd=f"{working_dir}/{workspace}")
+
+    # Copy the config files we need
+    config_url = f"https://github.com/{owner}/{repo}/raw/main/files/config.json.example"
+    config_response = requests.get(config_url)
```

Comment:
Is there a reason this is downloaded instead of grabbed from the clone we just did? Same with sd-journalist.sec.example below.

Reply:
Another sort-of vestige of the original system. That's a good shout, I'll tweak it to use the files from the clone.